firebase-debug.log

```diff
@@ -0,0 +1,40 @@
+[debug] [2022-05-02T21:03:11.155Z] ----------------------------------------------------------------------
```

Comment:
Could we add the log files to the gitignore?

Reply:
Yup. Added.